akka-cluster-typed/src/test/scala/docs/akka/cluster/typed/BasicClusterExampleSpec.scala

```diff
@@ -0,0 +1,191 @@
+package docs.akka.cluster.typed
+
+import com.typesafe.config.ConfigFactory
+import org.scalatest.{ Matchers, WordSpec }
+//#cluster-imports
+import akka.actor.typed._
+import akka.actor.typed.scaladsl._
+import akka.cluster.ClusterEvent._
+import akka.cluster.MemberStatus
+import akka.cluster.typed._
+//#cluster-imports
+import akka.testkit.typed.scaladsl.TestProbe
+import org.scalatest.concurrent.{ Eventually, ScalaFutures }
+import org.scalatest.time.{ Millis, Seconds, Span }
+
+import scala.concurrent.duration._
+
+object BasicClusterExampleSpec {
+  val configSystem1 = ConfigFactory.parseString(
+    s"""
+#config-seeds
+akka {
+  actor {
+    provider = "cluster"
+  }
+  remote {
+    netty.tcp {
+      hostname = "127.0.0.1"
+      port = 2551
```

Comment:
not great to use fixed ports in our tests, I guess this is because it's included in docs.
perhaps worth overriding the config in the running test if that is not included in docs?